README.md

```diff
@@ -18,12 +18,12 @@ It relies on a few libraries:
 * [Boost](http://www.boost.org/) 1.40 or later.
   Required libraries: *thread*, *system*, *regex*, and *date_time*.
   Optional library for Boost Test driver: *test*.
-* [GTest](http://code.google.com/p/googletest/) 1.4 or later.
-  Optional for the GTest driver.
+* [GTest](http://code.google.com/p/googletest/) 1.6 or later.
```

Comment:
You said in #89 :

> Earliest supported must be those in the latest Ubuntu stable (Boost 1.54, Google Mock/Test 1.6.0), latest Debian stable (Boost 1.55, Google Mock/Test 1.7.0), ...

I've understood that 1.6.0 should be oldest supported. 
Haven't tested with older version, also haven't added to travis. But i don't see any reason the same code wouldn't work with older versions - just like stated in README:

>   It might work with earlier versions of the libraries, but it was not
 tested with them.